app/src/main/java/com/example/dungeontest/Settings.kt

```diff
@@ -206,7 +198,7 @@ fun SettingsScreen(
 }
 
 @Composable
-fun TextInputField(tokenValue: MutableState<TextFieldValue>, labelText: String) {
```

Comment:
if we're going to have one composable called TextInputField in one file and another with the same name in the other file, could you please make each one private (so it's only visible in its own file)?